client/executor/wasmi/src/lib.rs

```diff
@@ -295,9 +306,17 @@ impl<'a> wasmi::ModuleImportResolver for Resolver<'a> {
 			}
 		}
 
-		Err(wasmi::Error::Instantiation(
-			format!("Export {} not found", name),
-		))
+		if self.enable_stub {
+			trace!("Could not find function {}, a stub will be provided instead.", name);
+			self.missing_functions.borrow_mut().push(name.to_string());
```

Comment:
Please also create 1 unique index per function.

Reply:
done

Reply:
Ok so I think this is good but let me know